src/components/Editor/index.js

```diff
@@ -28,52 +28,50 @@ const styles = {
     }
 };
 
-const Editor = ({ modules, actions }) => {
-    return (
-        <div style={styles.editor}>
-            {modules.map((module, key) => (
-                <textarea
-                    rows={1}
-                    ref={el => 
-                        el && el.scrollHeight !== module.height ?
-                            actions.module.update({
-                                id: module.id,
-                                height: el.scrollHeight,
-                            }) :
-                            el
-                    }
-                    onChange={e =>
-                        e.currentTarget.value ?
-                            actions.module.update({
-                                id: module.id,
-                                value: e.currentTarget.value,
-                                height: 0
-                            }) :
+const Editor = ({ modules, actions }) => (
+    <div style={styles.editor}>
+        {modules.map((module, key) => (
+            <textarea
```

Comment:
Eventually, the Editor will support multiple modules (text, images, etc.) so I planned to make that abstraction down the line once I started down that road.  As of now, I was just leaving it within the Editor component itself since there wasn't any other logic yet.